mne/surface.py

```diff
@@ -1906,24 +1885,101 @@ def warp_montage_volume(montage, base_image, reg_affine, sdr_morph,
     return montage_warped, image_from, image_to
 
 
-def _get_neighbors(loc, image, thresh, voxels):
+_VOXELS_MAX = 100  # define constant to avoid runtime issues
+
+
+@fill_doc
+def get_montage_rois(montage, subject, subjects_dir=None,
+                     aseg='aparc+aseg', dist=5):
+    """Get the regions of interest near the channels in a montage.
+
+    .. note:: This is applicable for channels inside the brain
+              (intracranial electrodes).
+
+    Parameters
+    ----------
+    %(montage)s
+    %(subject)s
+    %(subjects_dir)s
+    %(aseg)s
+    dist : float
+        The distance in mm to use for identifying regions of interest.
+
+    Returns
+    -------
+    rois : dict
+        The regions of interest within ``dist`` of each channel.
+    colors : dict
+        The Freesurfer lookup table colors for the rois.
```

Comment:
Please check of the acronym ROI is used anywhere in mne and use caps for an acronym when not a variable name.

Reply:
It's used here https://github.com/mne-tools/mne-python/blob/main/examples/visualization/roi_erpimage_by_rt.py but it looks maybe an EEGLAB thing to mean general regions of sensor space (also several places here https://github.com/mne-tools/mne-python/blob/main/mne/channels/channels.py).

Also here in matlab.rst

```
 +--------------------------------+--------------------------------------------------------------+
 | mne_read_label_file            | Read a label file (ROI).                                     |
 +--------------------------------+--------------------------------------------------------------+
 | mne_write_label_file           | Write a label file (ROI).                                    |
 +--------------------------------+--------------------------------------------------------------+
```

And finally also the one that I assume you were thinking of is in several locations, which is using it synonymously with label.

```
# load the anatomical ROI for comparison
anat_label = mne.read_labels_from_annot(subject, parc='aparc',
                                       subjects_dir=subjects_dir,
```

I'll just go ahead and change to `label`.